### ISSUE TITLE
High GPU Usage

### ISSUE THREAD

**Issue by SuperCrackBros** [bkacjios/m-overlay#178]
**Describe the bug**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]
 - Dolphin Version: [e.g. Slippi 2.2.1]

**To Reproduce**
1. Open game '....'

**Expected behavior**

**Screenshots**

**Debug console output**
```
Open the settings menu by pressing the escape key on your keyboard.
Click on 'Open config directory' and copy/paste the contents of 'console.log' here.
```
**Additional context**

**Comment by bkacjios** [comment]
If you run melee and m-overlay and enter the konami code via your controller (up, up, down, down, left, right, left, right, B, A, Startt) some rendering debug info should start to display. Mind sending a screenshot of that?

**Comment by SuperCrackBros** [comment]
> If you run melee and m-overlay and enter the konami code via your controller (up, up, down, down, left, right, left, right, B, A, start) some rendering debug info should start to display. Mind sending a screenshot of that? Use the d-pad for up/down/left/right by the way.

![Screenshot 2023-07-17 15-51-35](https://github.com/bkacjios/m-overlay/assets/139812131/e1d591a3-e5e9-443a-b54c-6cb2d4338559)

**Comment by bkacjios** [comment]
360 FPS is odd. Is your monitor 360hz by chance?

**Comment by SuperCrackBros** [comment]
> 360 FPS is odd. Is your monitor 360hz by chance?

Yes. Capped it at 144hz but still get the issue which is also odd.

**Comment by bkacjios** [comment]
Strange. I have a 144hz monitor, NVIDIA GTX 3070 TI (with the same driver version as you) and task manager shows m-overlay only using around 3% to 5% GPU. The debug render info looks pretty normal to me as well. I'm really not sure what's going on here.. Does closing OBS or disabling music help at all?

**Comment by SuperCrackBros** [comment]
> Strange. I have a 144hz monitor, NVIDIA GTX 3070 TI (with the same driver version as you) and task manager shows m-overlay only using around 3% to 5% GPU. The debug render info looks pretty normal to me as well. I'm really not sure what's going on here.. Does closing OBS or disabling music help at all?
> 
> ![image](https://user-images.githubusercontent.com/3247233/254093851-391d99eb-1dd2-4331-98d2-2ac449d49b38.png)

Still bugging out idk? Even reset PC to fix it earlier...
![Screenshot 2023-07-17 163017](https://github.com/bkacjios/m-overlay/assets/139812131/719a551c-7b77-4d03-b4c4-19973601ae60)

**Comment by bkacjios** [comment]
I'm curious if the new NVIDIA driver that just dropped (536.67) helps at all? I just have to assume this a 40 series issue, since both my 3070 and 2070 work fine.

**Comment by SuperCrackBros** [comment]
Updated driver & still high gpu usage atm
